.travis.yml

```diff
@@ -22,7 +15,7 @@ jobs:
     - stage: test
       script:
         - make fmt
-        - git diff --exit-code
+        - git diff --exit-code ':(exclude)go.sum'
```

Comment:
Can you share the motivation why we would want to exclude this? When we're checking in this file, then it seems to me that we should be checking whatever is produced by running `make deps && make fmt`.

Reply:
In short - as I can see, `go.sum` can be changed during `go build` (or `go <anything>`) in any way - entries may be added for... I don't know exactly what reason, it's not well-documented, possibly by changes to external repos, or while builing with different options, etc. So builds were not reproducible without ignoring changes to `go.sum` after `go <something>`. In pre-1.14, `go build` command even made changes to `go.mod`! Gonna check how things changed since then and try again (also look at this issue https://github.com/golang/go/issues/29492, `"... it gets populated differently under different scenarios..."`, `"In general, though, you can commit the go.sum file at any point: entries may be added or deleted as you run go mod tidy and various build/test commands, but the entry for a given version will not change."`)

So meaning of this `:(exclude)` is "we care just about source code after go fmt, but ignoring unstable go.sum file which can be changed for some reason during `go build / go fmt / go anything`".

Anyways - the main point is to check formatting of the source code. Any unstable files should be excluded.